src/wp-admin/themes.php

```diff
@@ -662,7 +669,9 @@
 			<td></td>
 		<?php } ?>
 	</tr>
-	<?php foreach ( $broken_themes as $broken_theme ) : ?>
+	<?php
+	foreach ( $broken_themes as $broken_theme ) :
```

Comment:
This doesn't look like a related change. Was this accidental, or included to tidy things up while we're here?

Reply:
This was a PHPCS flagged it as a formatting error, so I changed it. I don't know exactly how that happens...there's a lot of code in these older files that doesn't meet our current standards, and every once in a while something near a change will suddenly get flagged as failing.